CorsixTH/Src/th_gfx_common.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2009-2013 Peter "Corsix" Cawley and Edvin "Lego3" Linge
```

Comment:
You can delete Corsix and Lego3 out of this copyright, as it's "your" file :)

Reply:
And year should be 2020

Reply:
th_gfx.h is the common graphics file. I'm not sure why this is needed.

Reply:
Should be fixed now